Package.swift

```diff
@@ -178,6 +179,12 @@ let package = Package(
       exclude: ["CMakeLists.txt"]
     ),
 
+    .target(
+      name: "SwiftSyntaxMacroExpansion",
+      dependencies: ["SwiftSyntax", "SwiftSyntaxMacros"],
+      exclude: ["CMakeLists.txt"]
+    ),
+
```

Comment:
Since this is a new target, could you add a new `MARK` section for it? Each `MARK` section only contains the target + its test target right now

Reply:
Huh, I didn't realize that, thank you, will do.